README.md

```diff
@@ -1,5 +1,18 @@
-# license-list-XML
-This is a workspace for developing and implementing a new XML source format for the SPDX License List. This source format is (or will be) used to generate the authoritative file formats for licenses on the SPDX License List. These authoritative [target/destination formats](https://github.com/spdx/license-list-data) will remain the same for the time being.
+# SPDX License List Source Files
+The [SPDX License List](https://spdx.org/licenses/) is a list of common open-source licenses (and associated exceptions) used to easily and efficiently identify such licenses in SPDX documents, SPDX identifiers (e.g. in source code), package managers, or elsewhere.
```

Comment:
nit: "common open-source" -> "common, open-source"?  And " or" -> "and".

Reply:
we will use the wording that is used on the top of the main license list page, as this has been heavily vetted by various people and other community groups (I will update shortly)

Reply:
I like the "or" --> "and" but I think the sentence works better without the comma.